WooCommerce/Classes/Styles/Style.swift

```diff
@@ -70,10 +74,23 @@ class DefaultStyle: Style {
 
     /// Fonts!
     ///
+    static let maxFontSize              = CGFloat(28.0)
     let actionButtonTitleFont           = UIFont.font(forStyle: .headline, weight: .semibold)
     let alternativeLoginsTitleFont      = UIFont.font(forStyle: .subheadline, weight: .semibold)
+    let headlineSemiBold                = DefaultStyle
```

Comment:
Usually if a method has to be broken up, we try to only do so on the parameters, like this:

`let headlineSemiBold                = DefaultStyle.fontForTextStyle(.headline,
                                                                        weight: .semibold,
                                                                        maximumPointSize: DefaultStyle.maxFontSize)`